examples/react/basic-file-based-codesplitting/src/routeTree.gen.ts

```diff
@@ -1,134 +1,105 @@
-import { FileRoute, lazyFn, lazyRouteComponent } from "@tanstack/react-router"
+import { FileRoute, lazyFn, lazyRouteComponent } from '@tanstack/react-router'
 
-import { Route as rootRoute } from "./routes/__root"
-import { Route as LayoutImport } from "./routes/_layout"
-import { Route as IndexImport } from "./routes/index"
-import { Route as PostsPostIdRouteImport } from "./routes/posts.$postId/route"
-import { Route as PostsIndexImport } from "./routes/posts.index"
-import { Route as LayoutLayoutBImport } from "./routes/_layout/layout-b"
-import { Route as LayoutLayoutAImport } from "./routes/_layout/layout-a"
-import { Route as PostsPostIdDeepImport } from "./routes/posts_.$postId.deep"
+import { Route as rootRoute } from './routes/__root'
+import { Route as LayoutImport } from './routes/_layout'
+import { Route as IndexImport } from './routes/index'
+import { Route as PostsPostIdRouteImport } from './routes/posts.$postId/route'
+import { Route as LayoutLayoutBImport } from './routes/_layout/layout-b'
+import { Route as LayoutLayoutAImport } from './routes/_layout/layout-a'
+import { Route as PostsPostIdDeepImport } from './routes/posts_.$postId.deep'
 
-const PostsComponentImport = new FileRoute("/posts").createRoute()
-const LayoutLayoutBTestComponentImport = new FileRoute(
-  "/_layout/layout-b/test",
-).createRoute()
+const PostsComponentImport = new FileRoute('/posts').createRoute()
 
 const PostsComponentRoute = PostsComponentImport.update({
-  path: "/posts",
+  path: '/posts',
```

Comment:
why are the quotes changed? does this originate from https://github.com/TanStack/router/pull/948 ?

Reply:
I just run `pnpm run prettier` to format all files